### PR TITLE
Hide targets

### PR DESCRIPTION
**Pull request by kateherz**

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ var MyScene = new Phaser.Class({
         this.load.image('bg', '../assets/space.jpeg');
 
     },
-
+    target: [],
```
Comment (akundla) [inline scripts/main.js:33]: targets, not target

**Review by rapheh — DISMISSED** [review]
woah!

File: scripts/main.js [hunk 2]
```diff
@@ -42,15 +42,17 @@ var MyScene = new Phaser.Class({
...
-
+    
         for (let i = 20; i < gameWidth; i += 50) {
             for (let j = 50; j < gameHeight; j += 50) {
-                target[count] = this.add.image(i, j, "star");
-                target[count].angle = Math.floor(Math.random() * 90);
-                target[count].setInteractive();
-                target[count].visible = true;
+                this.target[count] = this.add.image(i, j, "star");
+                this.target[count].angle = Math.floor(Math.random() * 90);
```
Comment (akundla) [inline scripts/main.js:49]: this is a magic number, make it a constant

File: scripts/main.js [hunk 3]
```diff
@@ -98,7 +100,22 @@ var MyScene = new Phaser.Class({
             x = game.input.mousePointer.x;
             y = game.input.mousePointer.y
             console.log(x, y);
+            
+        }
+        starToTurnOn = Math.floor(Math.random() * this.target.length);
```
Comment (akundla) [inline scripts/main.js:105]: this is starToTurnOnIndex (because it is an index

File: scripts/main.js [hunk 4]
```diff
@@ -98,7 +100,22 @@ var MyScene = new Phaser.Class({
...
+        
+        twinkle(this.target[starToTurnOn]);//call the star function
+        for(let i=0;i<this.target.length;i++){
```
Comment (akundla) [inline scripts/main.js:108]: needs spaces for readability

File: scripts/main.js [hunk 5]
```diff
@@ -98,7 +100,22 @@ var MyScene = new Phaser.Class({
...
+            if(this.target[i].visible === true){
+               this.target[i].time+=1;
```
Comment (akundla) [inline scripts/main.js:110]: spaces, etc

File: scripts/main.js [hunk 6]
```diff
@@ -208,5 +227,15 @@ function Question (text, answer) {
     this.answer = answer;
 }
 
+function twinkle(object){
```
Comment (akundla) [inline scripts/main.js:230]: why so much whitespace?

**Review by akundla — CHANGES_REQUESTED** [review]
functinoality fine, needs formatting changes

**Comment by kateherz** [comment]
Comments have been resolved, please take another look. @akundla @rapheh

**Review by akundla — APPROVED** [review]
Thanks for making those changes. Approved for merge

**Review by CompCodeHub — APPROVED** [review]
Great work!!

**Pull request merged**